src/domain/community/virtualContributor/vcProfilePage/VCProfileContentView.tsx

```
@@ -1,146 +1,194 @@
+                      )}
+
+                      <Caption>{bullet.text}</Caption>
+                    </Gutters>
+                  </Gutters>
+                ))}
+
+                {cell.description && (
+                  <Gutters disableGap disablePadding>
+                    <Caption sx={{ textAlign: 'center' }}>
+                      {/* dangerouslySetInnerHTML is used temporarily because we're using hard-coded values. REMOVE when data is fetched from server and use Trans! */}
+                      <span dangerouslySetInnerHTML={{ __html: cell.description }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)